app/models/ability.rb

```diff
@@ -228,6 +228,7 @@ def checkout_permissions
       end
       can :return, Checkout, user: @user
       can :return_all, Checkout, user: @user
+      can :display_returned, Checkout, user: @user
```

Comment:
Instead of making a new action, I think we could reuse the `:read` action since it I think they are the same thing.  You would probably have to do something a little special in the controller though instead of adding this here.  I think this might be something like:
```
def display_returned
  authorize! :read, @checkouts
  ...
end
```
OR  see my comment in `config/routes.rb`